### ISSUE TITLE
[Bug]: Parse `DynArray` parameters that use values from imported modules

### ISSUE THREAD

**Issue by pcaversaccio** [benber86/mamushi#47]
Mamushi currently fails to parse functions that use values imported via modules for `DynArry` types; example:

```vyper
from .. import multicall as mc

@external
def multicall(data: DynArray[mc.Batch, mc._DYNARRAY_BOUND]) -> DynArray[mc.Result, mc._DYNARRAY_BOUND]:
    return mc._multicall(data)
```

will raise with:

```
error: cannot format multicall_mock.vy: Unable to parse input file, are you sure the Vyper code is valid?
Oh no! 💥 💔 💥
1 file failed to reformat.
```

**Comment by benber86** [comment]
Which version are you using? Could not reproduce with `0.0.4-a3`

```
(mamushi) [ben@arch-carbon temp]$ mamushi --version
mamushi, version 0.0.4-a3
(mamushi) [ben@arch-carbon temp]$ mamushi downloads/test.vy --in-place False
reformatted downloads/test.vy
from .. import multicall as mc


@external
def multicall(
    data: DynArray[mc.Batch, mc._DYNARRAY_BOUND]
) -> DynArray[mc.Result, mc._DYNARRAY_BOUND]:
    return mc._multicall(data)

All done! ✨ 🍰 ✨
```

**Comment by pcaversaccio** [comment]
Hmm I used `0.0.3` - do I need to use prerelease versions?

**Comment by pcaversaccio** [comment]
Yeah looks like it's fixed in `0.0.4a3` indeed! So will close accordingly.
